Main.sublime-menu

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "id": "preferences",
+        "children":
+        [
+            {
+                "id": "package-settings",
+                "children":
+                [
+                    {
+                        "caption": "SublimeKickAssemblerC64",
+                        "children":
+                        [
+                            {
+                                "command": "open_file",
+                                "args": {
+                                    "file": "${packages}/SublimeKickAssemblerC64/README.md",
+                                },
+                                "caption": "README"
+                            },
+                            { "caption": "-" },
+                            {
+                                "command": "open_file",
+                                "args": {
+                                    "file": "${packages}/SublimeKickAssemblerC64/KickAssembler (C64).sublime-settings",
+                                },
+                                "caption": "Settings – Default"
+                            },
+                            {
+                                "command": "open_file",
+                                "args": {"file": "${packages}/User/KickAssembler (C64).sublime-settings"},
+                                "caption": "Settings – User"
+                            },
+                            { "caption": "-" },
```

Comment:
I suggest removal of line 34-46, or I will do it later. I will merge `SublimeKickAssemblerC64/Preferences.sublime-settings` into `/SublimeKickAssemblerC64/KickAssembler (C64).sublime-settings`, don´t know why I put stuff in different files.

Reply:
If good idea, can also be done later, after merging PR.

Reply:
Never mind.